src/client/src/orangehrmPimPlugin/pages/employee/Employee.vue

```diff
@@ -105,12 +108,11 @@
           v-model:order="sortDefinition"
           :headers="headers"
           :items="items?.data"
-          :selectable="true"
-          :clickable="true"
+          :selectable="$can.delete('employee_list')"
+          :clickable="false"
```

Comment:
This will disable the clicking on row. we need to be able to click the row and navigate to profile

Reply:
@Ajiharan007 Please check this with 50-beta and let me know what change have caused this. the fix probably should apply from OXD Library side